app/controllers/api/v1/orders_controller.rb

```diff
@@ -40,7 +40,7 @@ def index
         return my_orders if is_browse_app?
         return recent_designations if params['recently_used'].present?
         records = @orders.with_eager_load.
-          search(params['searchText'], params['toDesignateItem'].presence).latest.
+          search(params['searchText'], params['toDesignateItem'].presence).descending.
```

Comment:
Place the . on the next line, together with the method name.  

code climate warning.